src/utils/chain.ts

```diff
@@ -15,18 +15,43 @@
 import cli from 'cli-ux';
 import { getClient } from '../client';
 
+let blockID: string;
+
 interface ObserveParams {
 	readonly label: string;
 	readonly height: number;
 	readonly liskCorePath: string;
 	readonly delay: number;
+	readonly isFinal: boolean;
 }
 
-export const getChainHeight = async (liskCorePath: string): Promise<number> => {
+export const getChainHeight = async (liskCorePath: string, isFinal: boolean): Promise<number> => {
 	const client = await getClient(liskCorePath);
```

Comment:
Please consider renaming this method to `getAPIClient`. Also, please consider creating two separate methods: `getCurrentFinalizedHeight` and `getCurrentChainHeight`.
```suggestion
	const client = await getAPIClient(liskCorePath);
```